examples/BART/BART_introduction.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    rng = np.random.RandomState(RANDOM_SEED)
</pre><p><code>np.random.default_rng</code> the RandomState is the legacy generator. According to numpy docs:</p><p> </p><blockquote>This class should only be used if it is essential to have randoms that are identical to what would have been produced by previous versions of NumPy.</blockquote><p> </p><p>reference: https://numpy.org/doc/stable/reference/random/legacy.html#legacy</p><p> </p><p>note: if we don't generate samples with numpy we can have only a random seed passed to the pymc.Model instances</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
using `np.random.default_rng` does not work with `pm.Model()` I can use an integer for that as a workaround as you suggested, but still  it will not work with `pm.bart.predict`. So I guess this is ok for know until we update pymc to work with the new generator?

Reply:
Yeah, didn't know the update on pymc side was still missing. Is there an issue I cpuld subscribe to to avoid asking people to change this when it's not yet possible?

Reply:
Not sure, @ricardoV94 probably knows.